app/models/server_puppet_data.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class ServerPuppetData
+  attr_reader :client
+
+  def initialize
+    @client = PuppetdbClient.new(Settings.PUPPETDB_ENDPOINT)
+  end
+
+  def self.run
+    new.load_data
+  end
+
+  def load_data
+    Server.find_each do |server|
+      server.update(ip: @client.ipaddress(server.fqdn))
```

Comment:
You can use the `attr_reader` thing the class already has so therefore you can just use `client`